### PR TITLE
updated dependencies

### PR DESCRIPTION
**Pull request by bguayante**

### CODE REVIEW

File: queries/results/.dir
Comment (jungshadow) [inline queries/results/.dir:1]: @bguayante Why the empty file?

**Review by jungshadow — CHANGES_REQUESTED** [review]
If the results directory is generated from running the program, should it be checked into the repo?

**Comment by bguayante** [comment]
> If the results directory is generated from running the program, should it be checked into the repo?

It throws a "file not found" error if the folder doesn't already exist. The folder must have been ignored during initial development so it slipped through, though I recall cloning and testing on a fresh install.

Perhaps a better way to approach it would be to add a line to check for the folder in the query script and create it if it doesn't exist.

**Comment by jungshadow** [comment]
Either that or you should have it in the instructions to create the folder. I'd generally leave it up to the app to create its own output folder and/or have it configurable.

**Comment by bguayante** [comment]
@jungshadow I added support for the user to specify the export location as an argument when running the script. If no argument is provided, it will default to `queries/results/`. In both cases, the directory will be created if it does not already exist.

File: queries/run_query.py [hunk 1]
```diff
@@ -14,23 +16,42 @@ def main():
     """
     Script to export validated JSON-LD data out of a RDF store.
     """
+
+    # Define location to which results json will be saved from arg. Defaults to ./results/ if no
+    # argument provided.
+    def parse_args():
+        parser = argparse.ArgumentParser()
+        parser.add_argument(
+            "--loc",
```
Comment (jungshadow) [inline queries/run_query.py:25]: `--output`, `--output-dir`, or `-o` tends to be the default for command line programs for output locations, but this is an improvement on the previous functionality and not a deal breaker for me.

**Review by jungshadow — APPROVED** [review]
Fine by me. Just one comment for future development.

**Pull request merged**